docs/contributing/organization/core-team.rst

```diff
@@ -0,0 +1,102 @@
+Sylius Core Team
+================
+
+The **Sylius Core Team** is the group of developers that determine the
+direction and evolution of the Sylius project. Their votes rule if the
+features and patches proposed by the community are approved or rejected.
+
+All the Sylius Core Team members are long-time contributors with solid technical
+expertise and they have demonstrated a strong commitment to drive the project
+forward.
+
+This document states the rules that govern the Sylius Core Team. These rules
+are effective upon publication of this document and all Sylius Core Team members
+must adhere to said rules and protocol.
+
+Core Team Organization
```

Comment:
Would it be an idea to assign someone with 'Plugin Lead' focussing mainly on the proper plugin points inside Sylius to allow for swift development of plugins. Next to this, maybe also someone / several people should be in contact with certain plugin developer to help them improve important plugins?